src/ui/page_scannow.c

```diff
@@ -59,11 +59,12 @@ typedef struct {
     lv_obj_t *img1;
 } channel_t;
 
-channel_t channel_tb[10];
-channel_status_t channel_status_tb[10];
+#define SCAN_CH_NUM 12
```

Comment:
Couldn't you declare this constant in a more global place and use it in osd.c and dm6302.c as well?